en.lproj/SGLocalizable.strings

```diff
@@ -28,31 +28,31 @@
 "Stories.Warning.Notice" = "%@ WILL BE ABLE TO SEE that you viewed their Story.";
 "Stories.Warning.NoticeStealth" = "%@ will not be able to see that you viewed their Story.";
 
-"Settings.Photo.Quality.Notice" = "Quality of outgoing photos and photo-stories";
-"Settings.Photo.SendLarge" = "Send large photos";
-"Settings.Photo.SendLarge.Notice" = "Increase the side limit on compressed images to 2560px";
+"Settings.Photo.Quality.Notice" = "Quality of uploaded photos and stories.";
+"Settings.Photo.SendLarge" = "Send Large Photos";
+"Settings.Photo.SendLarge.Notice" = "Increase the side limit on compressed images to 2560px.";
 
 "Settings.VideoNotes.Header" = "ROUND VIDEOS";
-"Settings.VideoNotes.StartWithRearCam" = "Start with rear camera";
+"Settings.VideoNotes.StartWithRearCam" = "Start with Rear Camera";
 
 "Settings.CustomColors.Header" = "ACCOUNT COLORS";
 "Settings.CustomColors.Saturation" = "SATURATION";
 /* Make sure to escape Percentage sign % */
-"Settings.CustomColors.Saturation.Notice" = "Set saturation to 0%% to disable account colors";
+"Settings.CustomColors.Saturation.Notice" = "Set to 0%% to disable account colors.";
 
-"Settings.UploadsBoost" = "Uploads boost";
-"Settings.DownloadsBoost" = "Downloads boost";
+"Settings.UploadsBoost" = "Upload Boost";
+"Settings.DownloadsBoost" = "Download Boost";
 "Settings.DownloadsBoost.none" = "Disabled";
 "Settings.DownloadsBoost.medium" = "Medium";
 "Settings.DownloadsBoost.maximum" = "Maximum";
 
-"Settings.ShowProfileID" = "Show profile ID";
-"Settings.ShowDC" = "Show DataCenter";
-"Settings.ShowCreationDate" = "Show chat creation date";
-"Settings.ShowCreationDate.Notice" = "The creation date may be missing for some chats.";
-"Settings.SendWithReturnKey" = "Send with \"return\" key";
-"Settings.HidePhoneInSettingsUI" = "Hide phone in settings";
-"Settings.HidePhoneInSettingsUI.Notice" = "Your number will be hidden in Settings UI only. Go to Privacy settings to hide it from others.";
+"Settings.ShowProfileID" = "Show Profile ID";
+"Settings.ShowDC" = "Show Data Center";
+"Settings.ShowCreationDate" = "Show Chat Creation Date";
+"Settings.ShowCreationDate.Notice" = "The creation date may be unknown for some chats.";
+"Settings.SendWithReturnKey" = "Send with “return” Key";
```

Comment:
I personally hate this type of quotes. Please, revert them and make sure to escape with backslash